### DISCUSSION TITLE
Easy onboarding.

### DISCUSSION THREAD

**Discussion by NF8C** [arendst/Tasmota#15670] [Ideas]
In summary I have made a feature request to Homeassistant to add an easy onboarding process to their app.  Here is my vision of the process.  The app would scan for WiFi access points.  The user would be prompted to select the access point of the device they want to set up.  It would then allow you select the access point you want to connect the Tasmotta device too.  The app would then get the ip of the new Tasmotta device after it connected to the home network and display it's set up page.  

I see this potentially going farther at a later point.  The app could help you select the device and enter prestored setup information for things like MQTT.

To get advances integration going we might have to make sure the HomeAssistant app know's the correct URL to add things like MQTT details.  It's even possible to eventually make it easy for users to serch for their device type although my hope is to make it easy for people buying preflased devices like those from cloudfree.

Thoughts?
Technical limitation?

Here is the request I made with HomeAsstant.
https://github.com/home-assistant/android/issues/2538

Sorry if anything isn't clear  I'm very tired but I had to get all this out.

**Comment by Jason2866** [comment]
Tasmota is a solution not made for a specific home automation hub. Since resources are limited there will no support for just a specific home automation solution. There is the mqtt autodiscovery function which works very well with Home Assistant. Home Assistant picks all Tasmota devices and the can be used without configuration "work" in Home Assistant.
For initial setup there is Tasmota WebInstaller which configures wifi right after Tasmota is flashed. No need for an extra app.
Feel free to do and provide a Android / iOs App for configuring Tasmota or some more Apps for onboarding to a specific Home Automation System. The few developers around Tasmota wont do (enough "work" with Tasmota).
